### PR TITLE
Poppler

### PR DESCRIPTION
**Pull request by hkctkuy**
Add poppler

### CODE REVIEW

File: projects/poppler/build_fuzzer.sh [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+        -Doss_fuzz=disabled \
+        -Dlibmount=disabled \
+        _builddir
+    ninja -C _builddir
+    ninja -C _builddir install
+    popd
+
+    pushd $SRC/libpng
+    autoreconf -fi
+    CPPFLAGS=-I$PREFIX/include LDFLAGS=-L$PREFIX/lib ./configure --prefix="$PREFIX" --disable-shared --disable-dependency-tracking
```
Comment (SweetVishnya) [inline projects/poppler/build_fuzzer.sh:134]: We should add sanitizer flags here

File: projects/poppler/build_fuzzer.sh [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+  -DCMAKE_INSTALL_PREFIX=$PREFIX
+
+export PKG_CONFIG="`which pkg-config` --static"
+make -j$(nproc) poppler poppler-cpp poppler-qt5
+if [ "$SANITIZER" != "memory" ]; then
+    make -j$(nproc) poppler-glib
+fi
+
+export CFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
+export CXXFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
```
Comment (SweetVishnya) [inline projects/poppler/build_fuzzer.sh:214]: CFLAGS and CXXFLAGS are already defined

Comment (SweetVishnya) [reply projects/poppler/build_fuzzer.sh:214]: same for other sh scripts

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/poppler/build_fuzzer.sh (hunk 2 above)
Comment (hkctkuy) [reply projects/poppler/build_fuzzer.sh:214]: I use "-fsanitize=fuzzer" here instead "-fsanitize=fuzzer-no-link" as it was in the begging

**Pull request merged**